.github/workflows/api-client-coverage.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node: ['14.x']
+        node: ['16.x']
```

Comment:
About @richterb remark, I wonder if we could use the following config in order to make the tests run on multiple NodeJS versions:

```
node-version: ['14.x', '16.x']
```

Reply:
As far as I can tell from the doc, yes this should work, everything has already been written in this action to allow for it, by using `matrix` (https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) and `matrix.node` as vars in the steps.